src/control/cmd/dmg/storage.go

```diff
@@ -41,11 +41,11 @@ const (
 
 // storageCmd is the struct representing the top-level storage subcommand.
 type storageCmd struct {
-	Prepare   storagePrepareCmd   `command:"prepare" alias:"p" description:"Prepare SCM and NVMe storage attached to remote servers."`
-	Scan      storageScanCmd      `command:"scan" alias:"s" description:"Scan SCM and NVMe storage attached to remote servers."`
-	Format    storageFormatCmd    `command:"format" alias:"f" description:"Format SCM and NVMe storage attached to remote servers."`
-	Query     storageQueryCmd     `command:"query" alias:"q" description:"Query storage commands, including raw NVMe SSD device health stats and internal blobstore health info."`
-	SetFaulty storageSetFaultyCmd `command:"setfaulty" alias:"sf" descrption:"Manually set the device state of an NVMe SSD to FAULTY."`
+	Prepare storagePrepareCmd `command:"prepare" alias:"p" description:"Prepare SCM and NVMe storage attached to remote servers."`
+	Scan    storageScanCmd    `command:"scan" alias:"s" description:"Scan SCM and NVMe storage attached to remote servers."`
+	Format  storageFormatCmd  `command:"format" alias:"f" description:"Format SCM and NVMe storage attached to remote servers."`
+	Query   storageQueryCmd   `command:"query" alias:"q" description:"Query storage commands, including raw NVMe SSD device health stats and internal blobstore health info."`
+	Set     setFaultyCmd      `command:"set" alias:"s" description:"Manually set the device state of an NVMe SSD to FAULTY."`
```

Comment:
The description and struct name don't seem to match the command. Shouldn't they be something more general like, setStateCmd and "Manually set the device state"?

Reply:
sure I will update to make more general.